### ISSUE TITLE
Unique Number-III

### ISSUE THREAD

**Issue by ishi0301** [HarshCasper/NeoAlgo#988]
## 🚀 Feature
Given an array that contains all elements occurring 3 times, but one occurs only once. Find that unique element.
Input  : arr[] = {6, 2, 5, 2, 2, 6, 6}
          
Output: 5

### Have you read the [Contributing Guidelines on Pull Requests]
Yes

## Motivation
I want to solve this issue as a part of CH20.

## Pitch
We can use bitwise AND to find the unique element in O(n) time and constant extra space.

## Assignees 

- @ishi0301: C-Plus-Plus
- @ritvij14 : Java
- @narayan2111 : Python
- @salonishah01: C
- @rutujadhanawade: C-Sharp

**Comment by ishi0301** [comment]
@HarshCasper @Abhijit2505 @iamrajiv @atarax665 @SKAUL05. I would like to implement this in C++ as a part of CH20.Please assign me this issue. Thank you!

**Comment by ritvij14** [comment]
I would like to try this in java for CH20.

**Comment by narayan2111** [comment]
I would like to work on this issue in python

**Comment by salonishah01** [comment]
I would like to try this in C

**Comment by dhruv-doshi** [comment]
I would like to solve this problem

**Comment by shraddhavp** [comment]
I would like to take this in python

**Comment by rutujadhanawade** [comment]
I would like to do this in C#

**Comment by narayan2111** [comment]
this issue is  assigned to me but my name is not showing in  Assignees @iamrajiv

**Comment by HarshCasper** [comment]
We are logging the scores for the <b>Contributor's Hack 2020</b>. If you have one or more than one PR merged during the program, make sure to fill this [form](https://forms.gle/jG7kDteT5EAz23hk7)

**Comment by ayushgaur03** [comment]
Hi...I would like to work on the java version of this issue....Can I do it??

**Comment by ritvij14** [comment]
@ayushgaur03 You can check the description above, I have already done it and my PR is merged, in any issue if the description has a language with someone's name written after it that means it has been already assigned to that person.
